nestjs/src/profile/profile.controller.ts

```diff
@@ -87,10 +90,17 @@ export class ProfileController {
     return new EndorsementDto(endorsement);
   }
 
+  @Get(':username/endorsement-data')
+  @ApiOperation({ operationId: 'getEndorsementData' })
+  @ApiResponse({ type: EndorsementDto })
```

Comment:
I believe we need some access guard for this, cause it exposes data to anyone

Reply:
That's right, as far as I know, that was the plan

Reply:
It could be not a problem if app code was not in open source, but in this case we should make sure that only publicly available data is exposed here, I am not sure that all the stuff here is public